### DISCUSSION TITLE
닥터 아카기의 로봇 시운전 부탁

### DISCUSSION THREAD

**Discussion by Rirbe** [wb1016/sb_korpatch#48] [unfinished]
Hey there, friendo! I'm Dr. Akaggy, and I deal in robots! So I have this experimental modular space mech in need of a test drive and my usual pilots are... Indisposed. Could you do me a favor and fly it through my test course?

Just talk to me when you're ready to go!

안녕, 친구! 난 아카기 박사야! 난 로봇을 취급해! 나는 시운전이 필요한 실험적인 모듈식 우주 장치를 가지고 있지, 그리고 내가 주로 조종하는 조종사들은... 몸이 아프다고 하네? 부탁 하나 들어줄 수 있어? 내 시험 코스를 통과해 주면 된다고!

갈 준비가 되면 얘기해
![20220508010948_1](https://user-images.githubusercontent.com/35803731/167262856-c582f74c-184a-42da-904e-9462cc208fa3.jpg)
!

Are you ready for me to deploy you to my mech test course? I built it in an abandoned freighter I stumbled upon.
내 메카 테스트 코스에 메카를 배치할 준비가 됬니? 메카는 우연히 만난 버려진 화물선에서 만들었어.
![20220508011902_1](https://user-images.githubusercontent.com/35803731/167262961-6e85b896-462c-4438-a85a-b16069763737.jpg)

**Comment by OABHs** [comment]
게임이 오래되고 유저들이 사용하는 표현이 대부분 굳어지면서 한글패치에서도 쉽게 그 번역명을 바꿀 수 없는 경우가 있습니다.

Mech의 경우 사전상의 뜻은 기계공이나 메커니즘을 줄여부르는 말이지만, SF소설이나 건담같은 소위 메카물으로 분류되는 창작물에서는 **탑승 가능한 기계 장비**라는 의미를 가집니다.

위 같은 의미일때 적어도 한국이나 일본에선 보통 mech라고 해도 mecha처럼 발음 하는 경우가 많은 것으로 압니다.

그러나 오랜시간동안 '메크'라고 불려서 가능하면 해당 번역명을 유지하는 편이 좋을 것 같습니다.


Are you ready for me to deploy you to my mech test course? 

너는 내가 메크 테스트 코스에 너를 배치할 준비가 되었니?
(for me는 의미상 주어)

'전장에/전선에 군대를/영웅을/하수인을 배치하다' 이런 식으로 쓰이지
일상 생활에서  '~에 너를 배치한다'라고는 쓸 일이 잘 없다보니 어색하게 느껴질 수도 있습니다.

그래서

'내 메크 테스트 코스에 배치될 준비 됐어?'
'내 메크 테스트 코스로 이동할 준비 됐어?'

정도가 될 것 같습니다.

I built it in an abandoned freighter I stumbled upon. 

게임상에서 메크는 주인공과 함께 이동하고, 코스가 버려진 화물선에 지어진것이므로

it은 메크가 아니라 코스로 보입니다.

**Comment by wb1016** [comment]
'내 메크 테스트 코스로 이동할 준비 됐어?'가 적당해보입니다.
